### PR TITLE
Add support link item in side bar

### LINKED ISSUE #660: Add "Support Kiwix" item to menu
Our Android App has a `Support Kiwix` element in its hamburger menu. Android-related donations constitute about a quarter of all the support we receive from individual users. 

Can we please add a similar link to https://support.kiwix.org to the iOS app?

supportus (1)

### PR DESCRIPTION
**Pull request by DobleV55**
Fixes #660

### CODE REVIEW

File: Support/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -263,7 +263,10 @@
 "enum.navigation_item.new" = "New";
 "enum.navigation_item.downloads" = "Downloads";
 "enum.navigation_item.settings" = "Settings";
+"enum.navigation_item.donations" = "Support Kiwix";
 "enum.search_result_snippet_mode.disabled" = "Disabled";
 "enum.search_result_snippet_mode.paragraph" = "First Paragraph";
 "enum.search_result_snippet_mode.sentence" = "First Sentence";
 "enum.search_result_snippet_mode.matches" = "Matches";
+
+"donations.url" = "https://kiwix.org/en/get-involved/";
```
Comment (BPerlakiH) [inline Support/en.lproj/Localizable.strings:272]: I am a bit torn on where to put this URL, to be honest.
We have this page on the website in a couple of languages, but that's not 100% mappable to iOS/macOS supported languages.

Things to consider:
- the nicest user experience would be to direct the user to the support page in the same language (if possible)
- it is a content / link, that maybe should not be handed over to external translators
- maybe we could use 1 generic URL, and the website could be tweaked (if not already) to pick the best possible locale for the user automatically (based on browser settings etc). Alternatively we could pass the device locale as a query param to a generic URL, that could map to the best language server side, which might be more maintainable in the long run
- we could try to map the client (mobile / macOS) device selected language to the closest URL, and fall back to English, but that might become a maintenance overhead over time
- we could default the url to: `https://kiwix.org/get-involved/`, and not worry too much about it, and take it out from translations

@kelson42 What do you think ?

Comment (DobleV55) [reply Support/en.lproj/Localizable.strings:273]: Hi, I understand the considerations regarding the URL placement. I propose maybe utilizing the Localizable.strings file for each language.

For instance, I can add the link for each language directly into its respective Localizable.strings file, such as:

The `Support/es.lproj/Localizable.strings` file for the Spanish version would have the following content: `'Support_URL' = 'https://kiwix.org/es/get-involved/';`

This approach ensures that users are directed to the support page in their preferred language. If there's a language without corresponding support on the website, I'll include the English version URL in that file.

Would this approach even work?

Comment (rgaudin) [reply Support/en.lproj/Localizable.strings:273]: @BPerlakiH just use a static URL and let the website handle the rest.

Comment (rgaudin) [reply Support/en.lproj/Localizable.strings:273]: https://kiwix.org/get-involved/ to be clear

**Review by BPerlakiH — DISMISSED** [review]

**Review by BPerlakiH — APPROVED** [review]
Please update the branch with rebase (after solving the conflicts).

**Comment by DobleV55** [comment]
Hi @BPerlakiH , I've completed the rebase process on the main branch with the 660-add-support... branch to incorporate the latest changes. Here are the steps I took:

1. Checked out the main branch and pulled the latest changes:
`git checkout main && git pull`
2. Rebased the main branch with the 660-add-support... branch:
`git rebase 660-add-support...`
3. Checked out the 660-add-support... branch and pulled any changes:
`git checkout 660-add-support... && git pull`
The output from the git pull command on the 660-add-support... branch indicates a successful fast-forward merge with no conflicts:
```
Updating 1fbc4be3..4c6206d0
Fast-forward
 App/App_macOS.swift                  | 2 +-
 Support/en.lproj/Localizable.strings | 2 +-
 SwiftUI/Patches.swift                | 1 -
 3 files changed, 2 insertions(+), 3 deletions(-)
Generate project files for Xcode.........................................Passed
```
Since no conflicts were encountered during the rebase process and the fast-forward merge was successful, there are no additional steps required. Finally, after pushing the changes, Git confirms that everything is up to date.

Should I proceed with any additional steps?

File: Support/en.lproj/Localizable.strings (hunk 1 above)
Comment (kelson42) [reply Support/en.lproj/Localizable.strings]: This is clearly the job of the Website, one statis URL our side.

**Comment by kelson42** [comment]
This PR is not ready and does not really do what is requested, see [my comment done a month ago](https://github.com/kiwix/apple/issues/660#issuecomment-1964825890). Make this code generic, and create a PR at `kiwix/apple-custom-apps` to add this link for the DWDS app, exactly like on the Android custom app.

**Comment by DobleV55** [comment]
Hi @kelson42 , I added the support Kiwix button that opens in a separate browser in both the Kiwix desktop app and the iOS mobile version. If this isn't satisfactory for you, please reassign the ticket, and let someone else handle it.

**Comment by kelson42** [comment]
@BPerlakiH I guess you will have to finish this PR :(

**Comment by BPerlakiH** [comment]
@kelson42 I've updated it with a branding solution similar to Android.

**Review by rgaudin — APPROVED** [review]
LGTM

**Comment by kelson42** [comment]
@BPerlakiH Can you please implement this with a PR for DWDS like on Android, to secure it works?

**Review by BPerlakiH — APPROVED** [review]

**Comment by BPerlakiH** [comment]
@kelson42 I've updated this to be the same as on Android.
I've also opened a PR under custom apps, to parse and document the new (optional) property:
https://github.com/kiwix/kiwix-apple-custom/pull/62

**Pull request merged**